crates/weaver_semconv/src/v2/entity.rs

```diff
@@ -0,0 +1,104 @@
+// SPDX-License-Identifier: Apache-2.0
+
+//! The new way we want to define entities going forward.
+
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+use crate::{
+    group::GroupSpec,
+    v2::{attribute::AttributeRef, CommonFields},
+};
+
+/// An EntityGroup defines a new entity.
+#[derive(Serialize, Deserialize, Debug, Clone, JsonSchema)]
+#[serde(deny_unknown_fields)]
+pub struct EntityGroup {
```

Comment:
Do we need to keep this Group suffix (EntityGroup, EventGroup, MetricGroup)?
It would seem clearer to me without this suffix.